docs/examples/random.md

```diff
@@ -0,0 +1,75 @@
+# opsdroid skill-random
```

Comment:
Let's have a more useful title here that describes what this page is about. Perhaps something like 'Creating a skill for making random chance decisions'.